main/src/ca/uwaterloo/flix/language/phase/Simplifier.scala

```diff
@@ -21,6 +21,7 @@ import ca.uwaterloo.flix.language.ast.Ast.BoundBy
 import ca.uwaterloo.flix.language.ast.{Purity, Symbol, _}
 import ca.uwaterloo.flix.language.dbg.AstPrinter._
 import ca.uwaterloo.flix.util.{InternalCompilerException, ParOps}
+import ca.uwaterloo.flix.language.phase.unification.Substitution
```

Comment:
Is this used?

Reply:
> Is this used?

Yea - line 195

Reply:
I am a bit confused. Is this doing some job that perhaps should have been done in the Monomorpher?

Reply:
I don't think so because the `Struct` type doesn't carry around field types. It only carries around tparams. It is only when we convert to monotype that we do the substitution - hence, in the simplifier.

Reply:
OK; I am still not 100%, but you can explain to me on monday?